testsuite/forge/src/backend/local.rs

```diff
@@ -179,6 +182,21 @@ impl LocalSwarm {
     }
 }
 
+#[derive(Debug)]
+pub enum HealthCheckError {
+    Crashed(::std::process::ExitStatus),
```

Comment:
Let's nix the ExitStatus for now, I don't think its going to be applicable to the k8s backend